content/en/blog/2023-12-07-flux-release-around-the-corner/index.md

```diff
@@ -0,0 +1,42 @@
+---
+author: kingdonb
+date: 2023-12-07 00:00:00+00:00
+title: Flux v2.2 release is just around the corner!
+description: "Flux v2.2.0: Major Helm improvements!"
+url: /blog/2023/12/flux-v2-2-release-around-the-corner/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+Now that Flux has reached General Availability, there are planned 3 minor releases per year. So, the maintainer team has been hard at work to create this Flux v2.2.0 release, which is slated to come out next week! Keep your eye out!
+
+While this next release addresses hundreds of issues and has many improvements that will be covered in more detail by a follow-up blog post, we want to highlight one crowning achievement of the next release: Flux helm-controller's reconciliation model has undergone a significant overhaul, this is the one we've all been waiting for!
+
+We hope many Flux users will rejoice in the improvements of the updated helm-controller, and that you'll all update to the latest release. This will help, in particular, with the issue of how the automatic recovery of Helm releases could occasionally get stuck in a pending state.
+
+Two new controls become available in the CLI (and as annotations) that can either [force](https://github.com/fluxcd/helm-controller/blob/64fed65148342578c1ed4b2155cd81852c54557a/docs/spec/v2beta2/helmreleases.md#forcing-a-release) or [reset](https://github.com/fluxcd/helm-controller/blob/64fed65148342578c1ed4b2155cd81852c54557a/docs/spec/v2beta2/helmreleases.md#resetting-remediation-retries) the `HelmRelease`.
```

Comment:
Maybe the best compromise here is to just not include the links, until the release is out, and then I'll add links in a follow-up commit. (This feedback right here is the reason why I thought it would actually be simpler to release this blog concurrent with the actual release, not ahead-of-time. How do we refer to these features without linking any docs?)

Of course we can just remove the links, if we're happy with the text. This is the pre-release blog, so I'm not sure how else we can make it comprehensive except to link to some docs which are not released yet.

We could try linking to pull requests instead of linking to docs that are unreleased? WDYT

Reply:
You refer to the features by providing a brief overview of what will be added or changed, or showcasing it. This would not require references to documentation.

I continue to feel that without doing any of this, the blog post is just void and doesn't warn people up in any way, nor would it tell people more than what is stated in the release notes. Which means I still do not get the point of publishing this in its current form.